pom.xml

```diff
@@ -51,6 +51,11 @@
     </repositories>
 
     <dependencies>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>20.1.0</version>
+        </dependency>
```

Comment:
The score should be provided. The annotation classes are not effective within the classpath. It's for Intellij IDEA.

Reply:
bruh what does that mean

Reply:
What do you mean? 🤔

Reply:
scope*

Reply:
I'm talking specifically about the annotations from Jetbrains, not other annotation libraries.